ar.json

```diff
@@ -0,0 +1,193 @@
+{
+  "name" : "العربية",
+  "live" : true,
+  "translations" : {
+    "meta" : {
+      "thForDate" : false,
+      "dateFormat" : "[year] [month] [date]",
+      "months" : [
+        "كانون الثاني",
+        "شباط",
+        "آذار",
+        "نيسان",
+        "أيّار",
+        "حُزيران",
+        "تمّوز",
+        "آب",
+        "أيلول",
+        "تشرين الأول",
+        "تشرين الثاني",
+        "كانون الأول"
+      ],
+      "types" : {
+        "backpack" : "حقيبة ظهر",
+        "emote" : "رقصة",
+        "glider" : "مظلة",
+        "emoji" : "رمز تعبيري",
+        "loading" : "شاشة التحميل",
+        "outfit" : "شخصية",
+        "pickaxe" : "فأس",
+        "skydive" : "تاثير القفز",
+        "umbrella" : "مظلة",
+        "spray" : "رش",
+        "toy" : "لعبة",
+        "pet" : "حيوان",
+        "music" : "موسيقى",
+        "wrap" : "جلد",
+        "bundle" : "حزمة",
+        "misc" : "متفرقات"
+      },
+      "rarity" : {
+        "legendary" : "أسطوري",
+        "epic" : "ملحمي",
+        "rare" : "نادر",
+        "uncommon" : "مميز",
+        "common" : "عادي"
+      },
+      "translationNotAvailable" : "للأسف لا توجد ترجمة متاحة لهذه الصفحة.",
+      "translationIncomplete" : "لم تتم ترجمة كل النصوص في هذه الصفحة"
+    },
+    "nav" : {
+      "home" : "الرئيسية",
+      "shop" : "المتجر",
+      "cosmetics" : {
+        "dropdown" : "القائمة",
+        "upcoming" : "قادمة",
+        "list" : "قائمة",
+        "png" : "png",
+        "gallery" : "المعرض",
+        "icons" : "الرموز",
+        "reminders" : "تذكير",
+        "history" : "تاريخ المتجر"
+      },
+      "news" : "الأخبار",
+      "modes" : "الأطوار",
+      "randomiser" : "randomiser",
+      "about" : {
+        "dropdown" : "حول",
+        "translate" : "الترجمة",
+        "about" : "عن fnbr.co",
+        "privacy" : "سياسة الخصوصة",
+        "donate" : "تبرع",
+        "discord" : "ديسكورد",
+        "twitter" : "تويتر"
+      },
+      "api" : "api",
+      "account" : {
+        "dropdown" : "الحساب",
+        "account" : "الحساب",
+        "logout" : "الخروج"
+      },
+      "languagePicker" : {
+        "trigger" : "اللغة",
+        "title" : "اختر اللغة",
+        "currentLanguage" : "أنت الآن تستخدم العربية."

```

Comment:
Please don't translate placeholders. `[language]` is missing.

Reply:
```suggestion
        "currentLanguage" : "أنت الآن تستخدم [language]."
```
like this?